Assets/PatchKit Patcher/Scripts/Version.cs

```diff
@@ -3,8 +3,8 @@ namespace PatchKit.Unity.Patcher
     public static class Version
     {
         public const int Major = 3;
-        public const int Minor = 10;
```

Comment:
This number is updated right before creating release candidate, no need to do it right now.